src/components/AddServer.js

```diff
@@ -22,9 +23,9 @@ export function AddServer({ isEditing }) {
         <AddIcon w={6} h={6} color="lightgray" />
       </Button>
       <AddServerModal isOpen={isOpen} onClose={onClose} />
-    </>
+    </Fragment>
   ) : (
-    false
+    null
```

Comment:
This fix was snuck in to avoid the possibility of false being rendered in place of the component when ```isEditing === false```